WISE-Community/WISE
# Implemented importing discussion work

Using "connected component", the discussion component can now import from multiple other discussion components. The student can continue to add new post and comment on posts from other components.

How to test
1. As a teacher, create the following project and set up a run:
step 1: discussion component
step 2: discussion component import work from step 1
step 3: discussion component import work from step 1 and step 2 (you'll need to add 2 connected components)

2. As multiple students working at different paces (e.g. student 1 can be on step 1, student 2 can be on step 2, etc):
   1. work on step 1. It should only show posts/comments that were made on step 1 by all students in the class. Even if you go to step 3 and come back to step 1, this should still be true.
   2. go to step 2. It should show all the students' posts/comments on step 1 and you can add new posts and comment on posts from step 1 and step 2.
   3. go to step 3. It should show all the students' posts/comments on step 1 and step 2 and you can add new posts and comment on posts from step 1, step 2 and step 3.

Closes #1873
